tools/depends/target/libdcadec/Makefile

```diff
@@ -0,0 +1,39 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=libdcadec
+VERSION=git-2a9186e3
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+# configuration settings
+
+LIBDYLIB=$(PLATFORM)/.libs/lib$(LIBNAME).a
+
+CLEAN_FILES=$(ARCHIVE) $(PLATFORM)
+
+CFLAGS += -std=gnu99
```

Comment:
not sure - but do we need this? From the upper layer of the depends buildsystem it normally uses c++11 standard (i don't know how the compiler behaves if both flags are set) - this might even be a problem on osx because the syntax is different ...